internal/flagv1/flag_pub_test.go

```diff
@@ -2,11 +2,12 @@ package flagv1_test
 
 import (
 	"fmt"
+	"testing"
```

Comment:
Can we keep the import as before, there is no change in them just a different order.